tools/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+repository="davrodpin/mole"
+install_path="/usr/local/bin"
+
+latest_version=$(curl --silent "https://api.github.com/repos/${repository}/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+
+# Get the os type
+os_type=$(uname -sm)
```

Comment:
You might want to review how this is handled.

`uname -sm` will give you both OS name and architecture.

The OS name should be covered to lower case
If the architecture is `x86_64`, then you should set as `amd64`, otherwise the script should fail, raising an error message to the user saying the architecture is not supported.